apps/web/components/Stepper/Stepper/Stepper.tsx

```diff
@@ -427,9 +440,19 @@ const Stepper = ({
       )
   }
 
+  // TODO: add webReaderClassName to things
```

Comment:
is this still a todo? things below seem to have webReaderClassName

Reply:
Sorry, it has been added. I removed the comment now.